src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -248,7 +248,10 @@ public static void ImportNoExponent()
 
             using (RSA rsa = RSAFactory.Create())
             {
-                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                if (rsa is RSACryptoServiceProvider)
+                    Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                else
+                    Assert.ThrowsAny<ArgumentException>(() => rsa.ImportParameters(imported));
```

Comment:
This feels fishy.  If everything worked before and you only changed RSACng I'd expect Linux and macOS to start failing.